docs/modules/migrate/pages/upgrading-from-imdg-3.adoc

```diff
@@ -2221,3 +2221,13 @@ See the following table for the before/after snippets.
 </hazelcast>
 ----
 |===
+
+== Deprecation of User Code Deployment
+
+The adding of Java classes to members using {ucd} was deprecated in Platform 5.4. 
+
+The Beta version of the {ucn} feature, which extends {ucd} to allow you to undeploy and redeploy your classes, is available for Enterprise users.
```

Comment:
> allow you to undeploy {...} your classes

A late-stage change is that [this isn't possible at run-time](https://hazelcast.atlassian.net/browse/HZ-4176), so maybe we should just focus on re-deploying?